32blit-sdl/Main.cpp

```diff
@@ -270,6 +296,13 @@ int main(int argc, char *argv[]) {
 	delete blit_capture;
 #endif
 
+    // delete game controllers
+    for (auto gc : blit_input->game_controllers)
+    {
+        SDL_GameControllerClose(gc.gc_id);
+    }
+    blit_input->game_controllers.clear();
+
```

Comment:
I'd say that should be in `Input`'s destructor... but it's currently not getting `delete`d. Something else to fix I guess...

Reply:
Indeed. Leaving there until fixed, or moving to the Input class and calling it from here?

Reply:
I'd just stick a delete with the other three below, fixing the (tiny) leak too. (Not that it matters much as we're exiting anyway...)